x-pack/docs/en/security/authentication/security-domain.asciidoc

```diff
@@ -0,0 +1,97 @@
+[role="xpack"]
+[[security-domain]]
+=== Security domains
+
+beta::[]
+
+Security domains are a method of grouping <<realms,realms>> to enable resource
+sharing across different realms for the same user.
```

Comment:
This wording "different realms for the same user" is a little awkward, but it's hard to come up with something better.

I think perhaps the problem is trying to describe it as "resource sharing" which is a little too much _implementation_. If we try to approach it from a conceptual point of view we can word things differently.
 
```suggestion
Security domains are a method of grouping <<realms,realms>> so that the {stack}
can recognize when a single user authenticates to two different realms.
```
I still don't love it, but I think it's easier to introduce the concept that way first.

Reply:
I modified your version a little bit to say
> Security domains are a method of grouping different <<realms,realms>>
so that the {stack} can recognize when a single user authenticates with
these realms.